src/widget/controlwidgetconnection.cpp

```diff
@@ -125,4 +126,6 @@ void ControlWidgetPropertyConnection::slotControlValueChanged(double v) {
         qDebug() << "Setting property" << m_propertyName
                 << "to widget failed. Value:" << parameter;
     }
+    pWidget->style()->unpolish(pWidget);
```

Comment:
A comment that explains why those 2 subsequent invocations are needed might help.

Reply:
Added.